crane-cloud/frontend
# Hide table for no transactions returned

# Description

This pull request hides table details when no transactions are found for a particular project.

## Type of change


## Trello Ticket ID

Please add a link to the Trello ticket for the task.

## How Can This Been Tested?

Run this branch locally and checkout the `fx-hide-table-details` to notice the change.


## Checklist:


# Screenshots

Before 
![before](https://user-images.githubusercontent.com/84501941/176931980-003a35ab-d8bd-4a14-8ca1-afe21a09974e.png)

After
![after](https://user-images.githubusercontent.com/84501941/176932036-9a1ae9b5-bb66-4bbe-81b2-d0fac4f8339c.png)
